meinberlin/apps/votes/templates/meinberlin_votes/token_export_dashboard.html

```diff
@@ -8,26 +8,26 @@ <h1 class="u-first-heading">{% translate 'Download voting codes' %}</h1>
     <p class="u-bottom-divider">{% blocktranslate %}Number of generated codes: {{ number_of_module_tokens }}.{% endblocktranslate %}</p>
     <ul class="u-list-reset">
         {% if token_export_url %}
-            {% for number, downloadable in token_packages %}
+            {% for package in token_packages %}
```

Comment:
maybe we could also do ```for package, downloadable, num_tokens``` here? I find it a bit hard to read..

Reply:
yes, much better, thanks